src/idl_gen_cpp.cpp

```diff
@@ -1187,6 +1191,21 @@ class CppGenerator : public BaseGenerator {
     }
   }
 
+  void GenTableCreateTemplate(StructDef &struct_def, std::string *code_ptr) {
```

Comment:
I agree for inline, but in this case other methods like GenTablePost(), GenTable(), etc. must be inlined as well, isn't it?